addon/session-stores/cookie.js

```diff
@@ -218,7 +229,7 @@ export default BaseStore.extend({
     let cookieOptions = {
       domain: this.get('cookieDomain'),
       expires: isEmpty(expiration) ? null : new Date(expiration),
-      path: '/',
+      path: this.get('cookiePath'),
```

Comment:
Hm, this is actually a breaking change. We'll need to default the property to `'/'` instead.